### PR TITLE
[test](show_data) test the correctness of data statistics in cloud mode

### PR DESCRIPTION
**Pull request by pingchunzhang**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary: add show data tests

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test

- Behavior changed:
    - [ ] No.

- Does this need documentation?
    - [ ] No.

### Check List (For Reviewer who merge this PR)

- [ ] Confirm test cases

### CODE REVIEW

**Review by dataroaring — DISMISSED** [review]
LGTM

File: regression-test/suites/show_data/ddl/lineitem_dup.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+CREATE TABLE IF NOT EXISTS lineitem_mow (
```
Comment (hello-stephen) [inline regression-test/suites/show_data/ddl/lineitem_dup.sql:1]: lineitem_dup ？

File: regression-test/plugins/aliyunOssSdk.groovy [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        /*
+            * Do not forget to shut down the client finally to release all allocated resources.
+            */
+        //client.shutdown();
+        logger.info("Done!")
+    }
+
+}
+
+// 获取某个存储空间下指定目录（文件夹）下的文件大小。
```
Comment (morningman) [inline regression-test/plugins/aliyunOssSdk.groovy:95]: English

**Comment by pingchunzhang** [comment]
run buildall

**Comment by pingchunzhang** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]

**Pull request merged**